README.md

```diff
@@ -97,14 +99,14 @@ $ xcaddy build \
     --with github.com/caddyserver/ntlm-transport@v0.1.1=../../my-fork
 ```
 
-You can even replace Caddy core using the `--with` flag:
+You can even replace Caddy core using the `--replace` flag:
 
 ```
 $ xcaddy build \
-    --with github.com/caddyserver/caddy/v2=../../my-caddy-fork
+    --replace github.com/caddyserver/caddy/v2=../../my-caddy-fork
     
 $ xcaddy build \
-    --with github.com/caddyserver/caddy/v2=github.com/my-user/caddy/v2@some-branch
+    --replace github.com/caddyserver/caddy/v2=github.com/my-user/caddy/v2@some-branch
 ```
```

Comment:
Well, technically it still works with `--with` cause we check for that. I think here it should show an example replacing a dep like `x/net` as you're doing, it's a better example.